frontend/src/components/c-title.vue

```diff
@@ -0,0 +1,31 @@
+<template lang="pug">
+div(v-html="markdownText")
+</template>
```

Comment:
We may have to set some padding/margins here so that the title is aligned correctly with the rest of the website.

<img width="1512" alt="Screenshot 2024-02-11 at 8 30 45 PM" src="https://github.com/reposense/RepoSense/assets/68203159/ec1a9921-e3bd-4578-8af8-30d2f8c110e7">

Reply:
I was thinking it that we could give the user complete freedom on the styling of their title component, but if we're primarily using this as a header for text, I do agree that the margins would make it look better.